tests/data/basic.json

```diff
@@ -1,2637 +1,2295 @@
-      "object_label": "xyxyxy",
-      "object_category": "biolink:AnatomicalEntity",
-      "subject_source": "a",
-      "object_source": "c",
-      "mapping_tool": "rdf_matcher",
-      "confidence": 0.83,
-      "comment": "mock data"
-    },
-    {
-      "subject_id": "c:something",
-      "predicate_id": "owl:equivalentClass",
-      "object_id": "b:something",
-      "match_type": [
-        "Lexical"
-      ],
-      "subject_label": "YXYXY",
-      "subject_category": "biolink:AnatomicalEntity",
-      "object_label": "yxyxyx",
-      "object_category": "biolink:AnatomicalEntity",
-      "subject_source": "c",
-      "object_source": "b",
-      "mapping_tool": "rdf_matcher",
-      "confidence": 0.84,
-      "comment": "mock data"
-    }
-  ],
-  "creator_id": "cjm",
-  "subject_source": "c",
-  "object_source": "b",
-  "mapping_tool": "https://github.com/cmungall/rdf_matcher",
-  "mapping_date": "2020-05-30",
-  "comment": "mock data",
-  "@type": "MappingSet",
-  "@context": {
-    "Orphanet": {
-      "@id": "http://www.orpha.net/ORDO/Orphanet_",
-      "@prefix": true
-    },
-    "dc": "http://purl.org/dc/terms/",
-    "dce": "http://purl.org/dc/elements/1.1/",
-    "dcterms": "http://purl.org/dc/terms/",
-    "linkml": "https://w3id.org/linkml/",
-    "neuroname": {
-      "@id": "http://braininfo.rprc.washington.edu/centraldirectory.aspx?ID=",
-      "@prefix": true
-    },
-    "oboInOwl": "http://www.geneontology.org/formats/oboInOwl#",
-    "oio": "http://www.geneontology.org/formats/oboInOwl#",
-    "owl": "http://www.w3.org/2002/07/owl#",
-    "pav": "http://purl.org/pav/",
-    "prov": "http://www.w3.org/ns/prov#",
-    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
-    "skos": "http://www.w3.org/2004/02/skos/core#",
-    "sssom": "http://w3id.org/sssom/",
-    "@vocab": "http://w3id.org/sssom/",
-    "author_id": {
-      "@type": "@id",
-      "@id": "pav:authoredBy"
-    },
-    "comment": {
-      "@id": "rdfs:comment"
-    },
-    "confidence": {
-      "@type": "xsd:double"
-    },
-    "creator_id": {
-      "@type": "@id",
-      "@id": "dc:creator"
-    },
-    "license": {
-      "@type": "@id",
-      "@id": "dcterms:license"
-    },
-    "mapping_cardinality": {
-      "@context": {
-        "@vocab": "@null",
-        "text": "skos:notation",
-        "description": "skos:prefLabel",
-        "meaning": "@id"
+   "mapping_set_id":"http://w3id.org/sssom/mapping/tests/data/basic.tsv",
+   "license":"https://creativecommons.org/publicdomain/zero/1.0/",
+   "mappings":[
+      {
+         "subject_id":"x:appendage",
+         "predicate_id":"owl:equivalentClass",
+         "object_id":"y:appendage",
+         "mapping_justification":"semapv:ManualMappingCuration",
+         "subject_label":"appendage",
+         "subject_category":"biolink:AnatomicalEntity",
+         "object_label":"appendages",
+         "object_category":"biolink:AnatomicalEntity",
+         "subject_source":"x",
```

Comment:
this should no longer be allowed: https://github.com/mapping-commons/sssom/blob/master/src/sssom_schema/schema/sssom_schema.yaml#L322

Reply:
Do not follow what you mean.

Reply:
subject_source is of type "EntityReference" - this here is a string ("x"), while it should be something like x:1.  This should not parse as a valid uri or curie!

Reply:
I dont understand how it can pass validation! It really should not.

Reply:
see https://github.com/mapping-commons/sssom-py/issues/265